src/Api/Controllers/UsersController.cs

```diff
@@ -16,7 +16,7 @@ public class UsersController : ApiControllerBase
 {
     private readonly ICurrentUserService _currentUserService;
 
-    public UsersController(ICurrentUserService currentUserService)
+    public UsersController(ICurrentUserService currentUserService, IMailService mailService)
```

Comment:
Why the fuck tho?
I don't u use it anywhere
so....yeh....
Ya no wut tahdu raii?